extensions/extensions-helm/aissemble-configuration-store-chart/values.template.yaml

```diff
@@ -27,8 +27,8 @@ aissemble-quarkus-chart:
         readOnly: true
     configMap:
       supplementalQuarkusConfig:
-        - quarkus.http.ssl.certificate.file=/etc/webhook/cert/tls.crt
-        - quarkus.http.ssl.certificate.key-file=/etc/webhook/cert/tls.key
+        - quarkus.http.ssl.certificate.files=/etc/webhook/cert/tls.crt
```

Comment:
Q: This config name changed in newer version of Quarkus?

Reply:
Correct